### PR TITLE
Fix template validation bug introduced in #346

### PR DESCRIPTION
**Pull request by ymmt2005**

### CODE REVIEW

File: cluster.go [hunk 1]
```diff
@@ -223,12 +222,17 @@ func (c *Cluster) Validate(isTmpl bool) error {
 	return nil
 }
 
-func (c *Cluster) validateNode(n *Node, isTmpl bool, fldPath *field.Path) error {
-	if !isTmpl {
+func validateNode(n *Node, isTmpl bool, fldPath *field.Path) error {
+	if isTmpl {
+		if len(n.Address) != 0 {
+			return errors.New("address is not empty: " + n.Address)
+		}
```
Comment (ymmt2005) [inline cluster.go:229]: why do you check this?
anyways, the address will be overwritten.

**Comment by ymmt2005** [comment]
@masa213f 
Please add `if err := c.Validate(false); err != nil ...` to this line
https://github.com/cybozu-go/cke/blob/master/sabakan/generator.go#L316

This makes the template instantiation super safe.

File: cluster.go (hunk 1 above)
Comment (masa213f) [reply cluster.go:229]: To check operation errors.

If the operator mistakenly set the CKE cluster definition (the result of `ckecli cluster get`) as a sabakan template, this check will cause an error.

**Comment by masa213f** [comment]
@ymmt2005 
Thanks. The comments are reflected.

**Comment by ymmt2005** [comment]
LGTM

**Pull request merged**